### PR TITLE
Create speech-bubble to contain About-Me Text

### PR DESCRIPTION
**Pull request by jessdejong**
Adding to the design of the About Me page by creating a speech bubble in CSS.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -10,19 +10,23 @@
...
-      I describe myself as a <strong>curious developer</strong>. In the future, I hope to become 
-      a <strong>decent ukulelist</strong>, a <strong>strong writer</strong>, and eventually, a 
-      <strong>full-time software developer</strong>.
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <div class="speech-bubble">
+        <div class="arrow"></div>
+
+        <p>If you're here, you probably already know that I'm a <strong>Google STEP intern</strong>.
```
Comment (estep-google) [inline portfolio/src/main/webapp/index.html:17]: nit: Paragraph formatting:

```
<p>
    Text ...
    More text ...
</p>
```

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -10,19 +10,23 @@
...
-      a <strong>decent ukulelist</strong>, a <strong>strong writer</strong>, and eventually, a 
-      <strong>full-time software developer</strong>.
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <div class="speech-bubble">
+        <div class="arrow"></div>
+
+        <p>If you're here, you probably already know that I'm a <strong>Google STEP intern</strong>. 
+        However, I'm also frequently described as an <strong>Incoming Sophomore at UT Austin</strong>,
```
Comment (estep-google) [inline portfolio/src/main/webapp/index.html:18]: nit: `<strong>` is not equivalent to `<b>` (bold tag). `<strong>` has more semantic meaning and is not often used. You should use `<b>` instead.

If we're getting _really_ fancy, you would put these in a `<span>` and style it via a CSS class using the `font-weight` property.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (jessdejong) [reply portfolio/src/main/webapp/index.html:17]: Thanks! I'll go ahead and fix this.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (jessdejong) [reply portfolio/src/main/webapp/index.html:18]: Thanks, I'll make sure to fix this and use <b> tags in the future.

Comment (jessdejong) [reply portfolio/src/main/webapp/index.html:18]: That was supposed to say < b>. Haha

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -10,19 +10,23 @@
...
+        However, I'm also frequently described as an <strong>Incoming Sophomore at UT Austin</strong>, 
+        a <strong>CS and Math Double Major</strong>, a <strong>fraternal twin</strong>, an 
+        <strong>amateur unicyclist</strong>, and a <strong>tennis enthusiast</strong>. When I work, 
+        I describe myself as a <strong>curious developer</strong>. In the future, I hope to become 
+        a <strong>decent ukulelist</strong>, a <strong>strong writer</strong>, and eventually, a 
+        <strong>full-time software developer</strong>.
+        <p>Click here to get a random message:</p>
+        <button onclick="addRandomGreeting()">Hello</button>
+        <div id="greeting-container"></div>
+
```
Comment (estep-google) [inline portfolio/src/main/webapp/index.html:31]: nit: Remove empty line

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -10,19 +10,23 @@
...
+      </div>
+
```
Comment (estep-google) [inline portfolio/src/main/webapp/index.html:33]: nit: Remove empty line

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -7,3 +7,35 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.speech-bubble {
+  background-color: #f8f8f8;
+  border: 2px solid #c8c8c8;
+  border-radius: 10px;
+  width: 500px;
```
Comment (estep-google) [inline portfolio/src/main/webapp/style.css:15]: nit: General CSS practice is to alphabetize properties

Comment (jessdejong) [reply portfolio/src/main/webapp/style.css]: Thanks! I'll keep that in mind in the future.

File: portfolio/src/main/webapp/style.css [hunk 6]
```diff
@@ -1,3 +1,5 @@
+/* Basic layout and overall site design */
+
```
Comment (estep-google) [inline portfolio/src/main/webapp/style.css:2]: Remove empty line

File: portfolio/src/main/webapp/style.css [hunk 7]
```diff
@@ -8,16 +10,20 @@
   margin-top: 20px;
 }
 
+/* Stylizing div as a speech bubble */
+
```
Comment (estep-google) [inline portfolio/src/main/webapp/style.css:14]: Remove empty line

File: portfolio/src/main/webapp/style.css [hunk 8]
```diff
@@ -8,16 +10,20 @@
...
-  width: 500px;
-  text-align:center;
   padding: 20px;
   position: absolute;
+  text-align:center;
+  width: 500px;
 }
 
+/* Adding arrow to speech bubble */
+
```
Comment (estep-google) [inline portfolio/src/main/webapp/style.css:26]: Remove empty line

**Comment by jessdejong** [comment]
Removed the blank lines

**Pull request merged**